pennylane/math/multi_dispatch.py

```diff
@@ -818,3 +818,21 @@ def expm(tensor, like=None):
     from scipy.linalg import expm as scipy_expm
 
     return scipy_expm(tensor)
+
+
+@multi_dispatch(argnum=[1])
+def gammainc(x, y, like=None):
+    """Return the lower incomplete Gamma function."""

```

Comment:
I think it would be worth to add some documentation here for the args x, y and return.

Reply:
Added.